src/app/dim-api/reducer.ts

```diff
@@ -1174,10 +1161,13 @@ function saveSearch(
   saved: boolean
 ) {
   const destinyVersion = account.destinyVersion;
-  const searchConfigs = searchConfigSelector(stubSearchRootState(account));
+  // Note: memoized
+  const filtersMap = buildFiltersMap(destinyVersion);
 
   // Canonicalize the query so we always save it the same way
-  const { canonical, saveable } = parseAndValidateQuery(query, searchConfigs);
+  const { canonical, saveable } = parseAndValidateQuery(query, filtersMap, {
+    customStats: draft.settings.customStats ?? [],
+  } as FilterContext);
```

Comment:
@nev-r was this (and the one below) simply forgotten about?

Reply:
not sure how you mean. pretty sure this is what it took to get it to believe custom string named stats were valid filters in all cases

Reply:
Apologies if it's unsure from the diff but right now that code doesn't pass any custom stat names here and I can't seem to save queries with custom stat names on beta.

Reply:
i see completely what you mean, thank you. same should have happened below.
it looks kinda control-z'd at some point, merging with master was really hectic.

Reply:
should be good in master

Reply:
actually maybe you mean something else with the original comment... saving custom stats is now working in master though